src/tools/rbd/action/Trash.cc

```diff
@@ -101,6 +101,9 @@ int execute_move(const po::variables_map &vm,
               << std::endl;
   }
 
+  if (expires_at != "now") {
+    std::cout << "rbd: image " << image_name << " will expire at " << exp_time << std::endl;
```

Comment:
Sorry for being late with this review. But what about the case when "expires_at" is not specified via command line arguments (i.e. when it is "now" by default)? I don't think we want to display this message in this case too.

Also, I am not sure we want to print this to stdout. This message looks to me more like a "log", and I would prefer to have it in stderr, though I don't have a strong opinion about this.

Reply:
Ah, it will most likely work as expected because `options->add_options()` sets the default value and `vm[EXPIRES_AT]` will always exist? The code looks a bit confusing but ok then.

Reply:
Default expiration time is "now".

Reply:
Initially I was confused by `if (vm.find(EXPIRES_AT) != vm.end()) {` condition above, which suggested to me that when the option was not specified you would have `vm.find(EXPIRES_AT) == vm.end()`, and in that case `expires_at` string would be empty. 

But it seems does not look like the case here, and if the option is not specified, we still have `vm.find(EXPIRES_AT) != vm.end()`. So the code is just confusing (the condition is not needed, or could be replaced with an assertion for safety), but it is not a problem of the current patch.

Reply:
@trociny I think the idea is to display time only when --expire-at is not now, for "now" case:
If we don't specify --expires-at, trash mv will output nothing, like before.

+1 for having it in logs from dev standpoint, but I am not sure from user perspective